client/hosting/sites/components/site-preview-pane/dotcom-preview-pane.tsx

```diff
@@ -96,7 +95,7 @@ const DotcomPreviewPane = ( {
 			},
 			{
 				label: __( 'Performance' ),
-				enabled: isActiveAtomicSite && config.isEnabled( 'performance-profiler/logged-in' ),
```

Comment:
@vykes-mac I think we want to revert that change.

Reply:
Why is that? we are able to access it from the url but the tab won't show. I think showing an imcomplete tab to other devs might not be a good experience ?

Reply:
I mean, the devs are aware the page is incomplete but they should be able to access it quickly w/o having to manually enter the URL. Not a big deal though, I thought this was committed by mistake.

Reply:
We can add it back when the UI start coming together 😆